napari/layers/image/image.py

```diff
@@ -176,6 +181,7 @@ def __init__(
         gamma=1,
         interpolation='nearest',
         rendering='mip',
+        render_as_plane=False,
```

Comment:
I think we should also take the plane as an argument. I think people will want to initialize the layer with a specific plane and it will be useful for serializing/deserializing the layer. I think we should either accept a Plane3D object or a dictionary that can be passed to Plane3D. What do you think?

Reply:
yeah this makes a lot of sense! I like the idea of having a `plane_parameters` dict rather than forcing knowledge of the `Plane3D` object on people

Reply:
Makes sense to me, we can also just start with this bool, and get to full serialization capability later